config/install/search_api.index.primary.yml

```diff
@@ -44,13 +39,21 @@ third_party_settings:
     term_modifiers:
       slop: 3
       fuzzy: 1
+      fuzzy_analyzer: true
     advanced:
       index_prefix: ''
       collection: ''
       timezone: ''
-id: content_index
-name: 'Basic Content Index'
-description: 'An example index to get started with Solr 8 searching.'
+    multilingual:
+      limit_to_content_language: false
+      include_language_independent: true
+      use_language_undefined_as_fallback_language: false
+      specific_languages:
+        en: '0'
+      use_universal_collation: false
+id: primary
+name: primary
```

Comment:
Are there any references to index/server names in docs that we need to update now as a result of them changing names?

Reply:
I'm loading the config installs at inception and referencing them by name in the robofile